docs/software/sciapps/quantumespresso.md

```diff
@@ -1,4 +1,66 @@
 [](){#ref-uenv-quantumespresso}
 # Quantum ESPRESSO
-!!! todo
-    complete docs
+
+Quantum ESPRESSO is an integrated suite of Open-Source computer codes for electronic-structure calculations and materials modeling at the nanoscale. It is based on density-functional theory, plane waves, and pseudopotentials:
+
+PWscf (Plane-Wave Self-Consistent Field)
+FPMD (First Principles Molecular Dynamics)
+CP (Car-Parrinello)
+
+## ALPS (GH200)
+### Setup
+Download the uenv image for QuantumESPRESSO:
+
+
+### List available images
+
+```bash
+uenv image find quantumespresso
+```
+
+### Pull the image of interest
+
+```
+uenv image pull quantumespresso/v7.4:v2
+```
```

Comment:
I think this might be redundant with the UENV documentation, and we don't want to have to change this if the `uenv` interface changes. In CP2K I added the following after the description of the software at the very top.

https://github.com/eth-cscs/cscs-docs/blob/dca32a4077d5f89f1dd9e08e5cc97f65d4d5a9cd/docs/software/sciapps/cp2k.md?plain=1#L13-L16

What do you think?